hawc/templates/base.html

```diff
@@ -108,5 +108,6 @@
     <script type="text/javascript" src="{% static 'vendor/htmx/1.6.1/htmx.min.js' %}"></script>
     {% render_bundle 'main' %}
     {% block extrajs %}{% endblock %}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/select2-bootstrap-theme/0.1.0-beta.10/select2-bootstrap.min.css">
```

Comment:
add this to our vendored files in static/css: https://cdnjs.com/libraries/select2-bootstrap-theme